core/server/models/plugins/pagination.js

```diff
@@ -130,6 +128,11 @@ pagination = function pagination(bookshelf) {
         /**
          * ### Fetch page
          * A `fetch` extension to get a paginated set of items from a collection
+         *
+         * We trigger two queries:
+         * 1. count query to know how many pages left (important: we don't attach any group/order statements!)
+         * 2. the actualy fetch query with limit and page property
```

Comment:
We aren't using `fetchPage` because it was only just added in the latest version of bookshelf ;)

By all means, we can refactor to use it instead?